src/js/components/Booking.js

```diff
@@ -159,6 +173,64 @@ class Booking{
     thisBooking.dom.wrapper.addEventListener('updated', function(){
       thisBooking.updateDom();
     });
+    thisBooking.dom.allTablesWrapper.addEventListener('click', function(){
+      thisBooking.initTables();
+    });
+    thisBooking.dom.form.addEventListener('submit', function(event){
+      event.preventDefault();
+      thisBooking.sendOrder();
+    });
+  }
+  initTables(){
+    const thisBooking = this;
+
+    const clickedElement = event.target;
+    console.log(thisBooking.starters)
+
+    if(clickedElement.classList.contains('table')){
+      const tableNumber = clickedElement.getAttribute('data-table');
+      if(!clickedElement.classList.contains('booked') && !clickedElement.classList.contains('selected')){
+        for (let table of thisBooking.dom.tables){
+          table.classList.remove('selected');
+        }
+        thisBooking.pickedTable = tableNumber;
+        clickedElement.classList.add('selected');
+      } else if(clickedElement.classList.contains('selected')){
+        clickedElement.classList.remove('selected');
+        thisBooking.pickedTable = null;
+      } else {
+        alert('Sorry, but that seat has been taken.');
+      }
+    }
+  }
+  sendOrder(){
+    const thisBooking = this;
+
+    const url = settings.db.url + '/' + settings.db.booking;
+    const payload = {
+      date: thisBooking.datePicker.value,
+      hour: thisBooking.hourPicker.value,
+      table: parseInt(thisBooking.pickedTable),
+      duration: parseInt(thisBooking.hoursAmountWidget.value),
+      ppl: parseInt(thisBooking.peopleAmountWidget.value),
+      starters: [],
+      phone: thisBooking.dom.phone.value,
+      address: thisBooking.dom.address.value,
+    };
+    for(let starter of thisBooking.dom.starters) {
+      if(starter.checked === true){
```

Comment:
Tu z kolei chcesz sprawdzić czy jeśli checkbox jest zaznaczony tj. `truthy` (https://developer.mozilla.org/en-US/docs/Glossary/Truthy) - jeśli tak to wejdź w if'a. Zapis `if (starter.checked) {...}` w zupełności wystarczy.